src/Routing/Router.php

```diff
@@ -594,6 +594,8 @@ public static function url($url = null, $full = false)
                 unset($url['_ssl']);
             }
 
+            $url = static::_applyUrlFilters($url);
```

Comment:
This will mean that routes generated with names have very little to no context anymore.